computer_setup.md

```diff
@@ -91,13 +89,6 @@ Download the VARA setup from https://rosmodem.wordpress.com/. Put the setup zip
 
 Unzip: `unzip VARA\ FM\ v4.1.4\ Setup.zip`
 
-We appear to need to replace pdh.dll. This seems to be common. You can get a compatible version from K6ETA or OE7DRT's websites.
-```
-wget https://oe7drt.com/blog/2021-10-03-winlink-and-vara-on-linux/pdh.dll.zip
```

Comment:
(Note to self more than anything else: This has now moved to https://oe7drt.com/assets/files/pdh.dll.zip)